components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -1,16 +1,24 @@
 import React from "react";
 import Image, { StaticImageData } from "next/image";
 
-import JavascriptIcon from "img/icons/☕️.svg";
+import JavascriptIcon from "img/icons/👾.svg";
 import ReactIcon from "/img/icons/⚛️.svg";
 import PythonIcon from "/img/icons/🐍.svg";
 import AIIcon from "/img/icons/🤖.svg";
 import MLIcon from "/img/icons/🧠.svg";
 import RustIcon from "/img/icons/🦀.svg";
+import PhpIcon from "/img/icons/📄.svg";
```

Comment:
think we need an issue here to remove the icons that are also emojis, it's clearing a lot of technical debt on incremental improvements to this page